packages/range/range.0.8/opam

```diff
@@ -0,0 +1,23 @@
+opam-version : "2.0"
+synopsis: "Fold on integer range"
+description: """
+Library to work on range of sequential integers with features like : split, map,
+filter capacities.
+Designed for making distributed computing easier with libraries like functory.
+"""
+maintainer : "Aldrik KLEBER <contact@aldrik.net>"
+authors : "Aldrik KLEBER"
+license : "GPL-3.0-only"
+homepage : "https://github.com/aldrikfr/range"
+dev-repo: "git+https://github.com/aldrikfr/range"
+bug-reports :"https://github.com/aldrikfr/range/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "base" { >="v0.12" & <"v0.14"}
+  "dune" {>= "1.8"}
```

Comment:
This should match the `dune-project` once it's there.

Reply:
> This should match the `dune-project` once it's there.

I'm not sure of what you mean

Reply:
That the lower bound here reflects the version used in https://github.com/aldrikfr/range/blob/0.8/dune-project#L1

Reply:
It's fine now